### PR TITLE
check pod status  before netctl config,

### PR DESCRIPTION
**Pull request by rchirakk**
check these before netctl config, 
1. contiv pod status
2. netplugin REST endpoint

fixes fwdmode errors during install

Signed-off-by: Ranjith <rchirakk@users.noreply.github.com>

### CODE REVIEW

File: install/k8s/install.sh [hunk 1]
```diff
@@ -266,19 +266,21 @@ cp ./netctl /usr/bin/
...
-	if [ "$?" -eq "0" ]; then
-		break
-	fi
-	sleep 10
+for i in {0..150}; do
+	sleep 2
+	# check contiv pods
+	$kubectl get pods -n kube-system | grep -v "Running" | grep -q ^contiv  && continue
+	# check netplugin status
+	curl -s localhost:9090/inspect/driver | grep -wq FwdMode || continue
```
Comment (neelimamukiri) [inline install/k8s/install.sh]: This only checks the FwdMode on the master node. What happens if the worker node starts up later?

File: install/k8s/install.sh [hunk 2]
```diff
@@ -266,19 +266,21 @@ cp ./netctl /usr/bin/
...
+	netctl tenant ls >/dev/null 2>&1 || continue
+	break
 done
 set -e
 
 if [ "$fwd_mode" == "routing" ]; then
 	netctl global set --fwd-mode $fwd_mode
+	sleep 5 # for re-init to complete
```
Comment (neelimamukiri) [inline install/k8s/install.sh]: Can we add a get in addition to the sleep, statics sleeps usually just make it harder to find issues later.

File: install/k8s/install.sh (hunk 1 above)
Comment (rchirakk) [inline install/k8s/install.sh]: there is one more check above that verifies all contiv pods are in running state

**Comment by neelimamukiri** [comment]
This handles the case for our current 2-node installs - assuming the pods dont restart after the `kubectl get` and before the `netctl set`. Given helps demo environments and makes things a little more stable than before, we'll go ahead with this for now.
Long term plan is to rethink if the fwdMode is really a parameter to change, rather than a parameter that needs to be passed at initialization time and redesign the install appropriately.

**Pull request merged**